cylc/flow/parsec/validate.py

```diff
@@ -582,27 +585,43 @@ def _unquoted_list_parse(cls, keys, value):
         Examples:
             >>> list(ParsecValidator._unquoted_list_parse(None, '"1", 2'))
             ['1', '2']
-
+            >>> list(ParsecValidator._unquoted_list_parse(
+            ...     None, 'I, "A<x,y>", J, "<a, b>B"'))
+            ['I', 'A<x,y>', 'J', '<a, b>B']
         """
-        # http://stackoverflow.com/questions/4982531/
-        # how-do-i-split-a-comma-delimited-string-in-python-except-
-        # for-the-commas-that-are
 
-        # First detect multi-parameter lists like <m,n>.
-        if cls._REC_MULTI_PARAM.search(value):
```

Comment:
Running this regex before attempting to parse the list will mean we either reject or pass the entire list.